### ISSUE TITLE
[BUG] "nonModHub" badge applied to FS25 ModHub mods

### ISSUE THREAD

**Issue by dskaggs** [FSGModding/FSG_Mod_Assistant#275]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

FS25 mods downloaded via in-game ModHub are displaying the label "nonModHub". All 3 mods in the screenshot below were downloaded via the in-game ModHub.

![image](https://github.com/user-attachments/assets/8b85cee1-57b7-44d9-8416-5372c6d3f79e)

### Expected Behavior

Should correctly identify ModHub mods and hid the "nonModHub" label appropriately

### Steps To Reproduce

In Windows 11, using Mod Assistant v5.1.4 and Farming Simulator 25 (latest version purchased directly from Giants as of creation of this ticket), download one of more mods from the in-game ModHub, refresh the monitored folder in Mod Assistant and see the "nonModHub" label applied to the mod.

### Environment

```markdown
- Mod Assistant Version: 5.1.4
- OS: Windows 11
- Virtualized: No
```

**Comment by jtsage** [comment]
correct.  give me a couple more days on this one, I'm not set up to parse modhub for 25 just yet.

**Comment by dskaggs** [comment]
Appreciate the quick response. I figured it was something like that but thought I'd create the issue just in case. Thanks for all that you do!

**Comment by jtsage** [comment]
aye. I appreciate the issue actually, it will keep this on my radar and make sure I actually remember to do it.

**Comment by jtsage** [comment]
fixed in 5.1.5 (out soon), closing as complete
